state/modeluser_test.go

```diff
@@ -391,6 +391,34 @@ func (s *ModelUserSuite) TestModelsForUser(c *gc.C) {
 	c.Assert(st.Close(), jc.ErrorIsNil)
 }
 
+func (s *ModelUserSuite) TestDeadModelsForUser(c *gc.C) {
+	user := s.Factory.MakeUser(c, nil)
```

Comment:
The blocks of code here and in the other new test which assert that there is a single model are not needed as the functionality is already tested in TestModelsForUser (and the code is cut'n'paste). All that's really needed is
s.Factory.MakeUser()
... set dead or importing
... assert 0 models